.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+
+node_js:
+  - 10
+
+# Stop as soon as an error is trigerred
+matrix:
+  fast_finish: true
+
+# Caching node_modules folder for speed up build
+cache: npm
+
+before_install:
+  - sh preinstall.sh
+
+env:
+  global:
+    - NODE_ENV=production
+    
```

Comment:
Can you remove the unnecessary spaces here

Reply:
Mmmm these spaces are pretty standard in the travis configuration file. (https://docs.travis-ci.com/user/environment-variables#global-variables)

Reply:
At the EOF?

Reply:
![image](https://user-images.githubusercontent.com/11588822/47724881-73e95a00-dc4f-11e8-8bd9-fc32b044b420.png)

Reply:
Oh, sorry I did not saw those 😕

Reply:
Done in  9ac43f3.